de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Average_Product_Prices.sql

```diff
@@ -31,19 +15,23 @@ SELECT
 		WHERE att.M_AttributeSetInstance_ID = COALESCE(ol.M_AttributeSetInstance_ID, iol.M_AttributeSetInstance_ID, pp.M_AttributeSetInstance_ID)) 
 		AS Merkmal
 	, ic.qtyOrdered AS Menge
+	, iol.MovementQty AS Menge_Lieferung
 	, uom.uomsymbol AS Mengenenheit
-	, ic.PriceActual_Net_Effective AS Preis
-	,(CASE WHEN c.iso_code != 'CHF'
-		THEN ROUND(currencyConvert(ic.PriceActual_Net_Effective * iol.MovementQty * COALESCE (uconv.multiplyrate, 1)
+	, round(COALESCE( (SELECT avg(il.PriceEntered) from C_Invoice_Line_Alloc ila 
```

Comment:
to be on the safe side, please also check for IsActive

Reply:
Done